Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputModule.cs

```diff
@@ -191,8 +191,22 @@ protected MouseState GetMousePointerEventDataForMrtkPointer(PointerData pointerD
 
             // Reset the RaycastCamera for projecting (used in calculating deltas)
             Debug.Assert(pointer.Rays != null && pointer.Rays.Length > 0);
-            RaycastCamera.transform.position = pointer.Rays[0].Origin;
-            RaycastCamera.transform.rotation = Quaternion.LookRotation(pointer.Rays[0].Direction);
+
+            if (pointer.Controller != null && pointer.Controller.IsRotationAvailable)
```

Comment:
Ideally, we should be able to do something like this all the time (no special casing):
```
RaycastCamera.transform.position = pointer.Position; 
RaycastCamera.transform.rotation = pointer.Rotation;
```

Reply:
The pointer abstraction is most useful if we make the position and rotation in each implementation mean something similar so it can can be used in situations like this interchangeably.

Reply:
I actually tried that first, but then touch interactions broke. It's really helpful to try it yourself and see how it breaks, since it's a bit complicated. The Poke Pointer will set its ray cast to be in front of the touchable surface even when the finger tip is behind the surface. However, the position and rotation always follows the position/rotation of the finger tip.

How would we solve this problem? 

A thing I thought up was to change position + rotation of the pointer to match the ray that poke pointer is using, but I was wary to do that because I didn't want to break existing behavior.

Reply:
Very interesting point. It seems like it could make sense, in the future, differentiating between where the pointer _is_ and where the pointer _wants to be_. In the case of the poke pointer, the pointer would be considered to be on the surface of the UI element it is interacting with and would want to be where the finger tip is. We need to know where the pointer wants to be to drive things like push buttons.